### PR TITLE
Adds grenade pouches.

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
Adds a pouch that holds only grenades.
Marines get a mostly full one with 4 grenades if they choose to vend it.

## Why It's Good For The Game
Kind of feel it's odd this never existed in the first place. I feel it'll go fine.

## Changelog
:cl:
add: Added new grenade pouches to pfc vendors and in general.
balance: Added grenade pouches that only hold grenades.
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -4. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/game/objects/items/storage/pouch.dm [hunk 1]
```diff
@@ -304,7 +304,20 @@
...
-	fill_type = /obj/item/explosive/plastique
+	fill_type = /obj/item/explosive/grenade/frag/upp
+	fill_number = 4
+
+/obj/item/storage/pouch/grenade
+	name = "Grenade pouch"
+	desc = "It can contain grenades."
+	icon_state = "explosive"
+	storage_slots = 6
+	max_w_class = 3
```
Comment (psykzz) [inline code/game/objects/items/storage/pouch.dm:315]: You should use the define instead

Comment (Pariah919) [reply code/game/objects/items/storage/pouch.dm]: it's using it's parent define now.

**Comment by spookydonut** [comment]
That's a lot of grenades

File: code/game/objects/items/storage/pouch.dm [hunk 2]
```diff
@@ -304,7 +304,19 @@
...
+/obj/item/storage/pouch/grenade
+	name = "Grenade pouch"
+	desc = "It can contain grenades."
+	icon_state = "explosive"
+	storage_slots = 6
+	can_hold = list(
+		/obj/item/explosive/grenade)
+
+/obj/item/storage/pouch/grenade/slightlyfull
+	fill_type = /obj/item/explosive/grenade/frag
```
Comment (Rohesie) [inline code/game/objects/items/storage/pouch.dm:319]: Why the frag type, the non-standard  and better one?

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-6). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**